test/runners.jl

```diff
@@ -123,6 +123,31 @@ end
         end
     end
 
+    # Test that we get no warnings when compiling without linking and when building a shared lib with clang
+    @testset "Clang - $(platform)" for platform in platforms
+        mktempdir() do dir
+            is_broken = platform in [Platform("armv7l", "linux"; libc="musl"), Platform("armv6l", "linux"; libc="musl")]
```

Comment:
Windows _is_ broken, and arm-musl does have "error"/"warning" in the output.  Also, can we get the links to #248 and #262 in the comments, please? :slightly_smiling_face:

Reply:
That's what I'm doing ;|